apps/guardian/src-tauri/tauri.conf.json

```diff
@@ -42,7 +42,12 @@
                 "icons/128x128@2x.png",
                 "icons/icon.icns",
                 "icons/icon.ico"
-            ]
+            ],
+            "windows": {
+                "certificateThumbprint": "3f305698ee54726dbe5324778abbd84196676b7c",
```

Comment:
I think you need to import the certificate into the keystore in order to use the thumbprint, which failed to make work for the desktop wallet.
Do they provide some custom signing function so we can use smctl?

Reply:
There seems to be a [`signCommand` configuration for tauri.conf.json](https://v1.tauri.app/v1/api/config#windowsconfig.signcommand).
Maybe you can get it to do the same as we do for the desktop wallet https://github.com/Concordium/concordium-desktop-wallet/blob/main/scripts/customSign.js